packages/client/src/constants/LeaderboardConsts.ts

```diff
@@ -0,0 +1,2 @@
+export const TEAM_NAME = 'team_01';
+export const SCORE_FIELD_NAME = 'huggywuggyscore';
```

Comment:
```suggestion
export const SCORE_FIELD_NAME = 'huggywuggyscore';
```
```suggestion
export const SCORE_FIELD_NAME = 'huggy_wuggy_score';
```
чтобы было одинаково с team_01